doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -110,10 +122,71 @@ Interface Sharpening
     The :doc:`../../examples/multiphysics/dam-break/dam-break` example discussed the interface sharperning mechanism.
 
 
+
+Algebraic Interface Reinitialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* ``subsection algebraic interface reinitialization``: defines parameters used to reinitialize the interface in VOF simulations. Alike the interface sharpening, this aims to reduce numerical smudging of the phase fraction and redefine the interface more compactly.
+
+  .. note::
+    A subsection on algebraic interface reinitialization will be added shortly to the :doc:`Volume of Fluid (VOF) Method<../../../theory/multiphase/cfd/vof>` theory guide for additional details on this method.
+
+  * ``enable``: enable the algebraic interface reinitialization.
```

Comment:
I don't mind this being a boolean, but we should double check that only one re-init mechanism is on at a single time :)

Reply:
You're totally right! I'll add it!